### PR TITLE
[DEV-2681] Updates the verbiage on IDV Page and the Advanced Search Page

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**
Updates the verbiage for three IDV Award types on the IDV Page and the Advanced Search Page

**Technical details:**
Updated two data maps on related to the IDV page, the other to the advanced search page.

`Indefinite Delivery, Definite Quantity Contract`
`Indefinite Delivery, Requirements Contract`
`Indefinite Delivery, Indefinite Quantity (IDIQ) Contract`

should be replaced respectively with....

`Indefinite Delivery / Definite Quantity Contract`
`Indefinite Delivery / Requirements Contract`
`Indefinite Delivery / Indefinite Quantity (IDIQ) Contract`

**JIRA Ticket:**
[DEV-2681](https://federal-spending-transparency.atlassian.net/browse/DEV-2681)

The following are ALL required for the PR to be merged:
- [x] Code review
- [x] Design review
- [x] [API #1813](https://github.com/fedspendingtransparency/usaspending-api/pull/1813) merged 
- [x] Changes deployed via execution of manage script by OPS team ([JIRA Ticket](https://federal-spending-transparency.atlassian.net/browse/OPS-693))

### CODE REVIEW

File: src/js/dataMapping/search/awardType.js [hunk 1]
```diff
@@ -3,26 +3,21 @@
...
-    'A': 'Blanket Purchase Agreements (BPA) Calls',
-    'B': 'Purchase Orders (PO)',
-    'C': 'Delivery Orders (DO)',
-    'D': 'Definitive Contracts',
-    'E': 'Unknown Type',
-    'F': 'Cooperative Agreement',
-    'G': 'Grant for Research',
-    'S': 'Funded Space Act Agreement',
-    'T': 'Training Grant',
-    'IDV_A': 'Government-Wide Acquisition Contract (GWAC)',
```
Comment (jonhill13) [inline src/js/dataMapping/search/awardType.js]: Just want to confirm that we want to change 
`'IDV_A': 'Government-Wide Acquisition Contract (GWAC)',	
'IDV_B': 'Multi-Agency Contract, Other Indefinite Delivery Contract (IDC)',
'IDV_C': 'Federal Supply Schedule (FSS)',	
'IDV_D': 'Basic Ordering Agreement (BOA)',	
'IDV_E': 'Blanket Purchase Agreements (BPA)'` in `awardTypeCodes` to their respective versions in `longTypeDescriptionsByAwardTypes `

Comment (jonhill13) [reply src/js/dataMapping/search/awardType.js]: And if so, we might want to consider changing `idvAwardTypes` in `dataMapping/contracts/idvAwardtypes.js`.

**Comment by AGuyNamedMarco** [comment]
@MaxwellKendall Approved – thanks!

File: src/js/dataMapping/search/awardType.js (hunk 1 above)
Comment (MaxwellKendall) [reply src/js/dataMapping/search/awardType.js]: @jonhill13 thanks for catching this, we don't want to change those at all.

**Comment by MaxwellKendall** [comment]
@jonhill13 responded to your changes with a27cf3a.

**Pull request merged**